src/footer/Footer.js

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
 import './footer.scss';
 import { Navigation } from '../navigation/Navigation';
+import { GetGeolocation } from '../GetGeolocation/GetGeolocation';
```

Comment:
`import { GetGeolocation } from '../GetGeolocation/GetGeolocation';` do not use twice 'GetGeolocation'.
We have index.js with re-export to avoid this nested definition.

Reply:
done